File: Source/StringRelatedOperations/to_upper.c
void toUpperCase(char *s, char *result) {
    for(int i = 0; s[i]; i++){
        if(s[i] >= 'a' && s[i] <= 'z'){
            result[i] = s[i] - 'a' + 'A';
        }
        else {
            result[i] = s[i];
        }
    }
}

File: Source/StringRelatedOperations/to_upper.js
function toUpperCase(s) {
    let result = '';
    for(let i = 0; i < s.length; i++){
        let code = s.charCodeAt(i);
        if(code >= 97 && code <= 122){
            result += String.fromCharCode(code - 97 + 65);
        } else {
            result += s[i];
        }
    }
    return result;
}
